<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayedGame</class>
 <widget class="QWidget" name="PlayedGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(9, 93, 19);
background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="hitButton_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>345</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 163, 15);
color:rgb(0,0,0);
font: 14pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_canvas_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>150</width>
     <height>229</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 161, 255);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_name">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>65</y>
     <width>63</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>pod 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_name">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>65</y>
     <width>63</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>pod 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_name">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>65</y>
     <width>63</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>pod 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hitButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>345</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 163, 15);
color: rgb(0, 0, 0);
font: 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hitButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>345</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>163</green>
         <blue>15</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 163, 15);
color: rgb(0, 0, 0);
font: 14pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainScore_value">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 24pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>180</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
color: rgb(255, 255, 255);
font: 10pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>Time: oyunu başlat</string>
   </property>
  </widget>
  <widget class="QLabel" name="deck_image">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>475</y>
     <width>135</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/resources/res/cards/00.png) 0 0 0 0 stretch stretch;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="deck_counter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>271</width>
     <height>185</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(82, 42, 42);
color:rgb(255,255,255);
font: 24pt &quot;Algerian&quot;;

border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border: 5px solid rgb(124, 64, 64);
border-bottom-color: rgb(255, 167, 161);
border-left-color: rgb(255, 167, 161);
</string>
   </property>
   <property name="text">
    <string>      ALL DECK
          SIZE:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>225</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(252,174,1,255);
color: rgb(0, 0, 0);
font: 16pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>RESTART GAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaderBoardButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>225</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(252,174,1,255);
color: rgb(0, 0, 0);
font: 16pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>LEADER BOARD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takeCardButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>475</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 163, 15);
color: rgb(0, 0, 0);
font: 28pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 198, 109);
font: 12pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>get a Score!</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 198, 109);
font: 12pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>get a Score!</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 198, 109);
font: 12pt &quot;Algerian&quot;;
</string>
   </property>
   <property name="text">
    <string>get a Score!</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_score">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>385</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_score">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>385</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_score">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>385</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>225</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(252,174,1,255);
color: rgb(0, 0, 0);
font: 16pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>START GAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_canvas_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>150</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_canvas_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>150</width>
     <height>169</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);
color: rgb(255, 255, 255);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_canvas_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>150</width>
     <height>139</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 14, 18);
color: rgb(255, 255, 255);



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod1_canvas_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>150</width>
     <height>109</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 255, 8);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_canvas_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>150</width>
     <height>229</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 161, 255);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_canvas_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>150</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_canvas_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>150</width>
     <height>169</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);
color: rgb(255, 255, 255);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_canvas_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>150</width>
     <height>139</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 14, 18);
color: rgb(255, 255, 255);



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod2_canvas_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>150</width>
     <height>109</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 255, 8);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_canvas_1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>150</width>
     <height>229</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 161, 255);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_canvas_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>150</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_canvas_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>150</width>
     <height>169</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);
color: rgb(255, 255, 255);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_canvas_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>150</width>
     <height>139</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 14, 18);
color: rgb(255, 255, 255);



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pod3_canvas_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>150</width>
     <height>109</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 255, 8);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame_pod_1">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>95</y>
     <width>160</width>
     <height>239</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border: 5px solid rgb(124, 64, 64);
border-bottom-color: rgb(255, 167, 161);
border-left-color: rgb(255, 167, 161);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame_pod_2">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>95</y>
     <width>160</width>
     <height>239</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border: 5px solid rgb(124, 64, 64);
border-bottom-color: rgb(255, 167, 161);
border-left-color: rgb(255, 167, 161);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame_pod_3">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>95</y>
     <width>160</width>
     <height>239</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border: 5px solid rgb(124, 64, 64);
border-bottom-color: rgb(255, 167, 161);
border-left-color: rgb(255, 167, 161);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame_pod_4">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>470</y>
     <width>145</width>
     <height>185</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border: 5px solid rgb(124, 64, 64);
border-bottom-color: rgb(255, 167, 161);
border-left-color: rgb(255, 167, 161);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
